src/common/commonutils/CommandUtils.c

```diff
@@ -29,270 +31,200 @@ static int NormalizeStatus(int status)
     return newStatus;
 }
 
-static int SystemCommand(void* context, const char* command, int timeoutSeconds, CommandCallback callback, OsConfigLogHandle log)
-{
-    const int callbackIntervalSeconds = 5; //seconds
-    const int defaultCommandTimeout = 60; //seconds
+#define BUFFER_SIZE 1024
```

Comment:
>1024 [](http://example.com/codeflow?start=20&length=4)

Is this the most optimal buffer size, or how was the 1KB size chosen?

Reply:
There's no optimal buffer size here, it was chosen as a random power of 2 that seemed reasonable.